docs/content/docs/reference/telemetry.md

```diff
@@ -63,6 +63,7 @@ Keystone collects telemetry information in the form of two different types of da
 We refer to these two different reports, as “device telemetry” and “project telemetry” respectively.
 
 These reports are forwarded to [https://telemetry.keystonejs.com/](https://telemetry.keystonejs.com/), and are reported separately to minimize any correlation between them insofar as the timing and grouping of that data, that an otherwise combined report may have. We are collecting these two reports for different reasons, and thus have no need to associate them.
+We differentiate and record the type and version of reports from the URL used by Keystone.
```

Comment:
Moving forward, we'll use the `/2` version in the `endpoint` to differentiate reports too in the same way we differentiate using `/project` and `/device`.

This makes that abundantly transparent.